app/Http/Kernel.php

```diff
@@ -55,7 +55,7 @@ class Kernel extends HttpKernel
      *
      * @var array
      */
-    protected $routeMiddleware = [
+    protected $middlewareAliases = [
```

Comment:
To clarify, did the variable name change within Laravel or is this a personal preference?

Reply:
this is part of the upgrade guide https://laravel.com/docs/10.x/upgrade#middleware-aliases